test/src/sdk-harness/test_projects/abi_impl_methods_callable/Forc.toml

```diff
@@ -4,3 +4,6 @@ authors = ["Fuel Labs <contact@fuel.sh>"]
 entry = "main.sw"
 license = "Apache-2.0"
 implicit-std = false
```

Comment:
Can be removed because it gets overridden by the `core` in `[dependencies]`.

Reply:
I'd like to keep this one for correctness' sake in case we change the dependencies.